web/src/beta/features/Visualizer/Crust/SketchTooltip/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { Typography } from "@reearth/beta/lib/reearth-ui";
+import { SketchFeatureTooltip } from "@reearth/beta/utils/sketch";
+import { styled } from "@reearth/services/theme";
+import { FC } from "react";
+
+type SketchTooltipProp = {
+  sketchFeatureTooltip?: SketchFeatureTooltip;
+};
+
+const BOLD_WORDS = [
+  "Drag",
+  "Drop",
+  "Click",
+  "Left click",
+  "Right click",
+  "Double click",
+  "Select",
+  "Delete",
+  "ESC"
+];
+
+const formatText = (text: string) =>
+  text
+    .split(new RegExp(`(${BOLD_WORDS.join("|")})`, "g"))
```

Comment:
How to handle the strong words in Japanese with this solution?

Reply:
@airslice I was thinking to add the Japanese words also in BOLD_WORDS array... do you have any other suggestion?? for to handle this

Reply:
https://react.i18next.com/latest/trans-component This might be helpful.